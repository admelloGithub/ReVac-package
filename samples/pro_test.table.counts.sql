\set semicolon_hack=1
\go
select count(*) "tableinfo" from tableinfo;
select count(*) "project" from project;
select count(*) "db" from db;
select count(*) "dbxref" from dbxref;
select count(*) "cv" from cv;
select count(*) "cvterm" from cvterm;
select count(*) "cvterm_relationship" from cvterm_relationship;
select count(*) "cvtermpath" from cvtermpath;
select count(*) "cvtermsynonym" from cvtermsynonym;
select count(*) "cvterm_dbxref" from cvterm_dbxref;
select count(*) "cvtermprop" from cvtermprop;
select count(*) "dbxrefprop" from dbxrefprop;
select count(*) "organism" from organism;
select count(*) "organism_dbxref" from organism_dbxref;
select count(*) "organismprop" from organismprop;
select count(*) "pub" from pub;
select count(*) "pub_relationship" from pub_relationship;
select count(*) "pub_dbxref" from pub_dbxref;
select count(*) "pubauthor" from pubauthor;
select count(*) "pubprop" from pubprop;
select count(*) "feature" from feature;
select count(*) "featureloc" from featureloc;
select count(*) "feature_pub" from feature_pub;
select count(*) "featureprop" from featureprop;
select count(*) "featureprop_pub" from featureprop_pub;
select count(*) "feature_dbxref" from feature_dbxref;
select count(*) "feature_relationship" from feature_relationship;
select count(*) "feature_relationship_pub" from feature_relationship_pub;
select count(*) "feature_relationshipprop" from feature_relationshipprop;
select count(*) "feature_relprop_pub" from feature_relprop_pub;
select count(*) "feature_cvterm" from feature_cvterm;
select count(*) "feature_cvtermprop" from feature_cvtermprop;
select count(*) "feature_cvterm_dbxref" from feature_cvterm_dbxref;
select count(*) "feature_cvterm_pub" from feature_cvterm_pub;
select count(*) "synonym" from synonym;
select count(*) "feature_synonym" from feature_synonym;
select count(*) "analysis" from analysis;
select count(*) "analysisprop" from analysisprop;
select count(*) "analysisfeature" from analysisfeature;
select count(*) "phylotree" from phylotree;
select count(*) "phylotree_pub" from phylotree_pub;
select count(*) "phylonode" from phylonode;
select count(*) "phylonode_dbxref" from phylonode_dbxref;
select count(*) "phylonode_pub" from phylonode_pub;
select count(*) "phylonode_organism" from phylonode_organism;
select count(*) "phylonodeprop" from phylonodeprop;
select count(*) "phylonode_relationship" from phylonode_relationship;
