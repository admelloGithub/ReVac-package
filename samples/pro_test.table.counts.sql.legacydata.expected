 tableinfo  
 -----------
           0

(1 row affected)
 project    
 -----------
           0

(1 row affected)
 db         
 -----------
          41

(1 row affected)
 dbxref     
 -----------
      110983

(1 row affected)
 cv         
 -----------
          15

(1 row affected)
 cvterm     
 -----------
       24148

(1 row affected)
 cvterm_relationship
 -------------------
               30126

(1 row affected)
 cvtermpath 
 -----------
           0

(1 row affected)
 cvtermsynonym
 -------------
          8289

(1 row affected)
 cvterm_dbxref
 -------------
         16644

(1 row affected)
 cvtermprop 
 -----------
        4100

(1 row affected)
 dbxrefprop 
 -----------
           4

(1 row affected)
 organism   
 -----------
           3

(1 row affected)
 organism_dbxref
 ---------------
               4

(1 row affected)
 organismprop
 ------------
            6

(1 row affected)
 pub        
 -----------
           1

(1 row affected)
 pub_relationship
 ----------------
                0

(1 row affected)
 pub_dbxref 
 -----------
           0

(1 row affected)
 pubauthor  
 -----------
           0

(1 row affected)
 pubprop    
 -----------
           0

(1 row affected)
 feature    
 -----------
      158179

(1 row affected)
 featureloc 
 -----------
      230520

(1 row affected)
 feature_pub
 -----------
           0

(1 row affected)
 featureprop
 -----------
      133408

(1 row affected)
 featureprop_pub
 ---------------
               0

(1 row affected)
 feature_dbxref
 --------------
         293969

(1 row affected)
 feature_relationship
 --------------------
                86168

(1 row affected)
 feature_relationship_pub
 ------------------------
                        0

(1 row affected)
 feature_relationshipprop
 ------------------------
                        0

(1 row affected)
 feature_relprop_pub
 -------------------
                   0

(1 row affected)
 feature_cvterm
 --------------
          21535

(1 row affected)
 feature_cvtermprop
 ------------------
              55013

(1 row affected)
 feature_cvterm_dbxref
 ---------------------
                     0

(1 row affected)
 feature_cvterm_pub
 ------------------
                  0

(1 row affected)
 synonym    
 -----------
           0

(1 row affected)
 feature_synonym
 ---------------
               0

(1 row affected)
 analysis   
 -----------
           6

(1 row affected)
 analysisprop
 ------------
            3

(1 row affected)
 analysisfeature
 ---------------
          129409

(1 row affected)
 phylotree  
 -----------
           0

(1 row affected)
 phylotree_pub
 -------------
             0

(1 row affected)
 phylonode  
 -----------
           0

(1 row affected)
 phylonode_dbxref
 ----------------
                0

(1 row affected)
 phylonode_pub
 -------------
             0

(1 row affected)
 phylonode_organism
 ------------------
                  0

(1 row affected)
 phylonodeprop
 -------------
             0

(1 row affected)
 phylonode_relationship
 ----------------------
                      0

(1 row affected)
